lib/streamlit/config.py

```diff
@@ -351,7 +351,7 @@ def _global_unit_test() -> bool:
         - 'arrow': Serialize DataFrames using Apache Arrow. Much faster and
           more efficient. This will become the default serialization format
           in the future.""",
-    default_val="legacy",
```

Comment:
"This will become the default serialization format in the future"...this change makes it default, right? Seems like the description could be updated to reflect this.

Reply:
Oops 🤦